cypress/fixtures/marriage/expected-no-records.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const search = {
+  systemNumber: '',
+  forenames: 'Winston',
+  surname: 'Churchil',
+  day: '30',
```

Comment:
Would prefer the date to look like this

dom: {
day: '29',
month: '02',
year: '2012'
}

and change subsequent usage accordingly.

Reply:
Done